### PR TITLE
chore(docs): Aqua 0.13.0 changelog

### PR DESCRIPTION
**Pull request by InversionSpaces**

### CODE REVIEW

File: docs/aqua-book/changelog.md [hunk 1]
```diff
@@ -7,6 +7,20 @@ Aqua compiler's versioning scheme is the following: `0.BREAKING.ENHANCING.RELEAS
 * `ENHANCING` part is incremented for every syntax addition
 * `RELEASE` is the release number, shows internal compiler changes, bugfixes that keep the language untouched
 
+## [0.13.0](https://github.com/fluencelabs/aqua/releases/tag/aqua-v0.13.0) - November 22, 2023
+
+* BREAKING CHANGE: **compiler:** Make `nil` of option of bottom type ([LNG-279](https://linear.app/fluence/issue/LNG-279)) ([#968](https://github.com/fluencelabs/aqua/issues/968))
```
Comment (alari) [inline docs/aqua-book/changelog.md]: What does it mean?

File: docs/aqua-book/changelog.md [hunk 2]
```diff
@@ -7,6 +7,20 @@ Aqua compiler's versioning scheme is the following: `0.BREAKING.ENHANCING.RELEAS
...
+* **compiler:** Always generate `last` argument of `fold` ([LNG-265](https://linear.app/fluence/issue/LNG-265)) ([#947](https://github.com/fluencelabs/aqua/issues/947))
+* **compiler:** Generate empty calls to `responseHandlerSrv` ([LNG-286](https://linear.app/fluence/issue/LNG-286)) ([#979](https://github.com/fluencelabs/aqua/issues/979))
+* **compiler:** Fix generation of wrong stream name in AIR ([LNG-276](https://linear.app/fluence/issue/LNG-276)) ([#958](https://github.com/fluencelabs/aqua/issues/958))
+* **compiler:** Fix incorrect focus on error ([LNG-274](https://linear.app/fluence/issue/LNG-274)) ([#959](https://github.com/fluencelabs/aqua/issues/959))
+* **compiler:** Fix multiple closures in one function ([LNG-262](https://linear.app/fluence/issue/LNG-262)) ([#941](https://github.com/fluencelabs/aqua/issues/941))
+* **compiler:** Fix comparing an alias and a named type with the same name ([LNG-231](https://linear.app/fluence/issue/LNG-231)) ([#946](https://github.com/fluencelabs/aqua/issues/946))
+* **compiler:** Fix that unknown service method call is ignored ([LNG-273](https://linear.app/fluence/issue/LNG-273)) ([#957](https://github.com/fluencelabs/aqua/issues/957))
```
Comment (alari) [inline docs/aqua-book/changelog.md]: What was changed?

File: docs/aqua-book/changelog.md (hunk 1 above)
Comment (InversionSpaces) [reply docs/aqua-book/changelog.md]: It means that literal value `nil` has type `?bottom`. If you have any suggestions on how to rephrase this, feel free to make a suggestion.

File: docs/aqua-book/changelog.md (hunk 2 above)
Comment (InversionSpaces) [reply docs/aqua-book/changelog.md]: Rephrased descriptions so that they explain what is their effect

File: docs/aqua-book/changelog.md (hunk 1 above)
Comment (alari) [reply docs/aqua-book/changelog.md]: > Breaking change: `nil` type changed from ???WHAT??? to `?⟂`, see THIS PAGE to learn about how to update your code

**Review by alari — APPROVED** [review]

**Pull request merged**